portfolio/src/main/webapp/style.css

```diff
@@ -86,3 +86,7 @@ h3 {
   height: 24px;
   width: 24px;
 }
+
+form {
+    padding: 0px;
```

Comment:
Nit: Use 2 spaces for indent

See https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
Done.